packages/tizen_package_manager/lib/tizen_package_manager.dart

```diff
@@ -18,6 +18,9 @@ enum PackageType {
   /// Tizen web/hybrid application package.
   wgt,
 
+  /// Tizen resource package.
+  rpk,
```

Comment:
This is an API change, so you need to update plugin version. probably update to 0.3.0.